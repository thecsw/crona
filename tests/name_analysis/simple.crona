glob:bool;

foo:int(a : int) {
	a = 10;
	return a;
}

main:int(){
	a:int;
	glob = a;
	foo(a);
	//b = a;
}
