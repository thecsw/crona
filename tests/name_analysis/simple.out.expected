glob(bool):bool;
foo(int->int):int(a(int):int){
	a(int) = 10;
	return a(int);
}
main(->int):int(){
	a(int):int;
	glob(bool) = a(int);
	foo(int->int)(a(int));
}
